src/vm.rs

```diff
@@ -900,10 +900,21 @@ impl<T: GuestStagePageTable> Vm<T, VmStateFinalized> {
         &self,
         csr_addr: u64,
         csr_len: usize,
-        _cert_addr: u64,
-        _cert_len: usize,
+        cert_addr: u64,
+        cert_len: usize,
         active_pages: &ActiveVmPages<T>,
     ) -> sbi::Result<u64> {
+        // Random ed25519 key pair.
+        let key_pair: [u8; 64] = [
+            239, 85, 17, 235, 167, 103, 34, 62, 7, 10, 32, 146, 113, 39, 96, 174, 3, 219, 232, 166,
+            240, 121, 167, 13, 98, 238, 122, 116, 193, 114, 215, 213, 175, 181, 75, 166, 224, 164,
+            140, 146, 53, 120, 10, 37, 104, 94, 136, 225, 249, 102, 171, 160, 97, 132, 15, 71, 35,
+            56, 0, 74, 130, 168, 225, 71,
+        ];
+
+        // Fake device identifier (DICE CDI)
+        let cdi: &[u8] = b"THISISARANDOMCDI";
```

Comment:
Where will the key pair and CDI eventually come from? Should there be a comment / TODO explaining that these are temporary along with what the long-term plan is?

Reply:
Good point, I added some explanatory comments.